ajax/libs/jquery.webticker/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "jquery.webticker",
+  "filename": "jquery.webticker.min.js",
+  "version": "3.0.0",
+  "description": "JQuery Web Ticker",
+  "keywords": [
+    "animation",
+    "ecosystem",
+    "ecosystem-jquery",
+    "horizontal",
+    "horizontal-scroll",
+    "image",
+    "image-scroll",
+    "jquery",
+    "jquery-plugin",
+    "marquee",
+    "news",
+    "news-ticker",
+    "plugin",
+    "scroll",
+    "ticker"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mazedigital/Web-Ticker.git"
+  },
+  "author": "Jonathan Mifsud",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/mazedigital/Web-Ticker.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jquery.webticker*"
+        ]
+      }
+    ]
+  },
+  "homepage": "https://maze.digital/webticker/",
+  "license": "ISC"
```

Comment:
Please update licenses to `GPL-3.0` and commercial license like #12297.

Reply:
Should we add commercial license to our license list file in separate commit?

Reply:
No, it's fine to add it in the same commit because it's a part of library adding.